Source/WebKit/Shared/SerializedTypeInfo.h

```diff
@@ -32,9 +32,14 @@
 
 namespace WebKit {
 
+struct SerializedType {
+    ASCIILiteral memberType;
+    ASCIILiteral memberName;
+};
+
 struct SerializedTypeInfo {
     ASCIILiteral name;
```

Comment:
How is memberName different than name?

Reply:
If we declare:

```
class X {
    int a;
};
```

SerializedTypeInfo::name == "X"
SerializedType::memberType == "int"
SerializedType::memberName == "a"